test/CMakeLists.txt

```diff
@@ -81,3 +81,30 @@ if (SINGULARITY_BUILD_PYTHON AND SINGULARITY_TEST_PYTHON)
            COMMAND ${Python_EXECUTABLE} ${CMAKE_CURRENT_SOURCE_DIR}/python_bindings.py -v)
   set_tests_properties(PythonBindings PROPERTIES ENVIRONMENT "${PYTHON_TEST_ENVIRONMENT}")
 endif()
+
+add_executable(profile_eos profile_eos.cpp)
+target_link_libraries(profile_eos singularity-eos::singularity-eos)
+
+if(SINGULARITY_USE_EOSPAC AND SINGULARITY_TEST_SESAME AND NOT SINGULARITY_USE_CUDA)
```

Comment:
This may be my own ignorance, but do we need to exempt CUDA or can it operate on the non-Kokkos code for the test?

Reply:
I remember I had to do this because it was failing the regit CI otherwise.